cosmos/launchpad/account.go

```diff
@@ -16,7 +17,11 @@ func (l launchpad) AccountBalance(ctx context.Context, request *types.AccountBal
 		return nil, ErrEndpointDisabledOfflineMode
 	}
 
-	resp, err := l.cosmos.GetAuthAccount(ctx, request.AccountIdentifier.Address)
+	var reqHeight string
+	if request.BlockIdentifier != nil {
+		reqHeight = strconv.Itoa(int(*request.BlockIdentifier.Index))
```

Comment:
Is easier if you set the client to use an integer I think, and it will make more sense than the string.

I think you can use 0 as no height set.

Reply:
yeah looks better, updated